[2022-03-01 21:49:54,739] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: dag_flights.read_process_write_file manual__2022-03-01T21:49:48+00:00 [queued]>
[2022-03-01 21:49:54,751] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: dag_flights.read_process_write_file manual__2022-03-01T21:49:48+00:00 [queued]>
[2022-03-01 21:49:54,751] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-01 21:49:54,751] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-01 21:49:54,751] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-01 21:49:54,769] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): read_process_write_file> on 2022-03-01 21:49:48+00:00
[2022-03-01 21:49:54,774] {standard_task_runner.py:52} INFO - Started process 2645 to run task
[2022-03-01 21:49:54,778] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag_flights', 'read_process_write_file', 'manual__2022-03-01T21:49:48+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/dag_flights.py', '--cfg-path', '/tmp/tmppg87x8kd', '--error-file', '/tmp/tmp5xwuyc47']
[2022-03-01 21:49:54,778] {standard_task_runner.py:77} INFO - Job 65: Subtask read_process_write_file
[2022-03-01 21:49:54,843] {logging_mixin.py:109} INFO - Running <TaskInstance: dag_flights.read_process_write_file manual__2022-03-01T21:49:48+00:00 [running]> on host 4a327ca97815
[2022-03-01 21:49:54,886] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/xcom.py:437: SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  return query.delete()

[2022-03-01 21:49:54,899] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/renderedtifields.py:126: SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  session.merge(self)

[2022-03-01 21:49:54,908] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/renderedtifields.py:162: SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
  tuple_(cls.dag_id, cls.task_id, cls.execution_date).notin_(subq1),

[2022-03-01 21:49:54,914] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/renderedtifields.py:163: SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
  ).delete(synchronize_session=False)

[2022-03-01 21:49:54,919] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_flights
AIRFLOW_CTX_TASK_ID=read_process_write_file
AIRFLOW_CTX_EXECUTION_DATE=2022-03-01T21:49:48+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-01T21:49:48+00:00
[2022-03-01 21:49:55,715] {__init__.py:490} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-_bnswh4_ because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-03-01 21:49:55,969] {font_manager.py:1443} INFO - generated new fontManager
[2022-03-01 21:51:09,149] {local_task_job.py:154} INFO - Task exited with return code Negsignal.SIGKILL
[2022-03-01 21:51:09,688] {taskinstance.py:1282} INFO - Marking task as FAILED. dag_id=dag_flights, task_id=read_process_write_file, execution_date=20220301T214948, start_date=20220301T214954, end_date=20220301T215109
[2022-03-01 21:51:09,755] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/dag.py:1079: SADeprecationWarning: Query.value() is deprecated and will be removed in a future release.  Please use Query.with_entities() in combination with Query.scalar() (deprecated since: 1.4)
  return qry.value(DagModel.is_paused)

[2022-03-01 21:51:09,755] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/dag.py:1079: SADeprecationWarning: Query.values() is deprecated and will be removed in a future release.  Please use Query.with_entities() (deprecated since: 1.4)
  return qry.value(DagModel.is_paused)

